addons/info/example/Button.js

```diff
@@ -14,6 +14,21 @@ Object.assign(Button, {
     style: PropTypes.object,
     disabled: PropTypes.bool,
     onClick: PropTypes.func,
+    array: PropTypes.array,
```

Comment:
I'd prefer if we could add example usages to the examples/cra-kitchen-sink.

The examples inside the packages are unofficially deprecated..